sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/reflection/KalixMethod.scala

```diff
@@ -95,11 +95,11 @@ case class VirtualServiceMethod(component: Class[_], methodName: String, inputTy
  * Build from methods annotated with @Subscription. Those methods are not annotated with Spring REST annotations, but
  * they become a REST method at the end.
  */
-case class RestServiceMethod(javaMethod: Method, optMethodName: Option[String] = None) extends AnyServiceMethod {
```

Comment:
not needed anymore. This was lingering after we decided not to pursue creating synthetic methods when adding @Subscribe.Topic. Instead we decided to forbid multiple @Subscribe.Topic for the same topic.